schema/rusk.proto

```diff
@@ -43,8 +36,16 @@ message GetProvisionersResponse {
     repeated Provisioner provisioners = 1;
 }
 
+message GetStateRootRequest {}
+
+message GetStateRootResponse {
+    bytes state_root = 1;
+}
+
 service State {
-    rpc VerifyStateTransition(VerifyStateTransitionRequest) returns (VerifyStateTransitionResponse) {}
+    rpc Echo(EchoRequest) returns (EchoResponse) {}
     rpc ExecuteStateTransition(ExecuteStateTransitionRequest) returns (ExecuteStateTransitionResponse) {}
```

Comment:
I expect to see `ExecuteStateTransition` accepting `finalized` field as it was suggested.

Reply:
`ExecuteStateTransitionRequest` included the attribute `finalized` so that is the case